README.md

```diff
@@ -1,83 +1,99 @@
 # aws-rds-infrastructure-cis-baseline
 
-InSpec Profile to validate the secure configuration of aws-rds-infrastructure-cis-baseline, against CIS's Amazon Web Services Three-tier Web Architecture Benchmark V1.0.0
+InSpec Profile to validate the secure configuration of aws-rds-infrastructure-cis-baseline, against [CIS'](https://www.cisecurity.org/cis-benchmarks/) Amazon Web Services Three-tier Web Architecture Benchmark V1.0.0
 
 ## Getting Started  
-It is intended and recommended that InSpec run this profile from a __"runner"__ host (such as a DevOps orchestration server, an administrative management system, or a developer's workstation/laptop) against the target remotely over __<transport_protocol>__.
+It is intended and recommended that InSpec run this profile from a __"runner"__ host (such as a DevOps orchestration server, an administrative management system, or a developer's workstation/laptop) against the target remotely over __ssh__.
 
+The latest versions and installation options are available at the [InSpec](http://inspec.io/) site.
 
-Latest versions and installation options are available at the [InSpec](http://inspec.io/) site.
-
-The following attributes must be configured in an attributes file for the profile to run correctly. More information about InSpec attributes can be found in the [InSpec Profile Documentation](https://www.inspec.io/docs/reference/profiles/).
+## Tailoring to Your Environment
+The following inputs must be configured in an inputs ".yml" file for the profile to run correctly for your specific environment. More information about InSpec inputs can be found in the [InSpec Profile Documentation](https://www.inspec.io/docs/reference/profiles/).
 
 ```
-# Description of input value
-input-name: default-value
-
 # Database instance identifier
 db_instance_identifier: []
 
 # VPC Security Group Id
-vpc_security_group_id: []]
+vpc_security_group_id: []
 
 # Allowed IP address range
 allowed_ip_address_range: ''
 ```
 
-## Usage
+## Configuring your AWS CLI
 
-InSpec makes it easy to run your tests wherever you need. More options listed here: [InSpec cli](http://inspec.io/docs/reference/cli/)
+The right system environment variables must be set with your AWS region and credentials and session token to use the AWS CLI and InSpec resources in the AWS environment. InSpec supports the following standard AWS variables:
 
 ```
-# Clone Inspec Profile
-$ git clone https://github.com/mitre/aws-rds-infrastructure-cis-baseline
-
 # Set required ENV variables
 $ export AWS_ACCESS_KEY_ID=key-id
 $ export AWS_SECRET_ACCESS_KEY=access-key
 $ export AWS_SESSION_TOKEN=session_token
 $ export AWS_REGION=us-west-1
```

Comment:
Added AWS CLI configuration steps in a separate section instead of being merged with the profile execution steps. Please let me know if I should modify it